ui/style.css

```diff
@@ -1729,6 +1729,20 @@ span.tag-input[contenteditable]:empty::before {
     animation: spin 2s linear infinite;
 }
 
+.leaderboard-loader {
+    margin-top: 5vh;
+    margin-left: auto;
+    margin-right: auto;

```

Comment:
This can actually be refactored to:
```suggestion
    margin: 5vh auto 0 auto;
```

Reply:
Did that thank you <3